gui/src/main/java/org/jboss/as/console/client/meta/CoreCapabilitiesRegister.java

```diff
@@ -38,123 +51,127 @@
     public static final String LOGGING_FORMATTER = "org.wildfly.logging.formatter";
     public static final String JGROUPS_STACK = "org.wildfly.jgroups.stack";
-    public static final String JGROUPS_CHANNEL = "org.wildfly.jgroups.channel";
-    public static final String UNDERTOW_CONNECTOR = "org.wildfly.undertow.connector";
+    public static final String JGROUPS_CHANNEL = "org.wildfly.clustering.jgroups.channel";
+    public static final String UNDERTOW_LISTENER = "org.wildfly.undertow.listener";
     public static final String UNDERTOW_SERVER = "org.wildfly.undertow.server";
     public static final String UNDERTOW_HOST = "org.wildfly.undertow.host";
     public static final String UNDERTOW_SERVLET_CONTAINER = "org.wildfly.undertow.servlet-container";
     public static final String UNDERTOW_RESPONSE_FILTER = "org.wildfly.undertow.filter";
 
-
+    private DispatchAsync dispatcher;
+    private HostStore hostStore;
     private final Capabilities capabilities;
 
     @Inject
-    public CoreCapabilitiesRegister(Capabilities capabilities) {
+    public CoreCapabilitiesRegister(DispatchAsync dispatcher, HostStore hostStore, Capabilities capabilities) {
+        this.dispatcher = dispatcher;
+        this.hostStore = hostStore;
         this.capabilities = capabilities;
     }
 
     @Override
-    //@SuppressWarnings({"HardCodedStringLiteral", "DuplicateStringLiteralInspection"})
     public void execute(final Control<BootstrapContext> control) {
 
-        // Selected capabilities from https://github.com/wildfly/wildfly-capabilities
-        capabilities.register(DATASOURCE, true,
-                AddressTemplate.of("/{selected.profile}/subsystem=datasources/data-source=*"));
-        
-        capabilities.register("org.wildfly.io.buffer-pool", true,
-                AddressTemplate.of("/{selected.profile}/subsystem=io/buffer-pool=*"));
-        
-        capabilities.register("org.wildfly.io.worker", true,
-                AddressTemplate.of("/{selected.profile}/subsystem=io/worker=*"));
-        
-        // Lists local and remote outbound socket bindings of all socket binding groups
-        capabilities.register(NETWORK_OUTBOUND_SOCKET_BINDING, true,
-                AddressTemplate.of("/socket-binding-group=*/local-destination-outbound-socket-binding=*"),
-                AddressTemplate.of("/socket-binding-group=*/remote-destination-outbound-socket-binding=*"));
-
-        // Lists all socket bindings of all socket binding groups
-        capabilities.register(NETWORK_SOCKET_BINDING, true,
-                AddressTemplate.of("/socket-binding-group=*/socket-binding=*"));
-
-        // Well-known capabilities not (yet) on https://github.com/wildfly/wildfly-capabilities
-        capabilities.register("org.wildfly.network.interface", true, AddressTemplate.of("/interface=*"));
+        ResourceAddress address = new ResourceAddress().add(HOST, hostStore.getSelectedHost());
```

Comment:
This didn't work in standalone mode. Please make sure to test for both standalone and domain mode. But never mind, I made the same mistake many time, as well ;-)